Source/Resource/Plugin/Language/eu.csv

```diff
@@ -11,46 +11,46 @@
  audio stream1; audio jarioa
  audio stream2; audio jarioa
  audio stream3; audio jarioa
- bit1; bit
- bit2; bit
- bit3; bit
- bps; bs-ko
- Bps; Bs-ko
- Byte1; Byte
- Byte2; Byte
- Byte3; Byte
- channel1; bide
- channel2; bide
- channel3; bide
- chapter1; atala
- chapter2; atal
- chapter3; atal
- chapters stream1; atal jarioa
- chapters stream2; atal jarioa
- chapters stream3; atal jarioa
- character1;
- character2;
- character3;
- day1; egun
- day2; egun
- day3; egun
- dB0;
- dB1; dB
- dB2; dB
- dB3; dB
- file1; agiri
- file2; agiri
- file3; agiri
- fps1; fs-ko
- fps2; fs-ko
- fps3; fs-ko
- frame1; frame
- frame2; frame
- frame3; frame
+ bit1; bit1

```

Comment:
Seems a regression, I doubt that you want "1" and so on displayed (several times, bit, Byte, channel...).
Remove the numbers.